lib/tasks/cloudhsm.rake

```diff
@@ -0,0 +1,6 @@
+namespace :cloudhsm do
+  desc 'Generate a new saml key'
+  task generate_saml_key: :environment do
+    CloudhsmKeyGenerator.new.generate_saml_key
```

Comment:
Does this print the label? How are you supposed to know this worked / what the generated key label is?

Reply:
That works, could you add an example transcript so we can see what that looks like?

Reply:
Sure.  Which form?  as comments on the rake task, or a separate standalone txt file.

Reply:
I mostly meant in this PR so we can see a sample of what it's supposed to look like, but at some point it would be good to have general documentation of the process since we're likely to be scratching our heads trying to remember how it works when we do this once a year.